phones/models.py

```diff
@@ -204,6 +204,23 @@ def relaynumber_post_save(sender, instance, created, **kwargs):
         )
 
 
+def last_inbound_date_default():
+    return datetime.now(timezone.utc)
+
+
+class InboundContact(models.Model):
+    relay_number = models.ForeignKey(RelayNumber, on_delete=models.CASCADE)
+    inbound_number = models.CharField(max_length=15)
```

Comment:
Suggestion: an index on `(relay_number, inbound_number)` for the query in `_get_phone_objects`. It's a light suggestion, since the index on `relay_number` should be OK for most users, and I expect incoming numbers in the dozens rather than the thousands, so a serial scan should be OK.